src/Cryptol/REPL/Command.hs

```diff
@@ -1993,61 +1993,6 @@ moduleInfoCmd isFile name
        rPutStrLn "}"
        pure emptyCommandResult
 
```

Comment:
Why did `extractCodeBlocks` get moved to `Cryptol.TypeCheck.AST`?  I don't think that's a good place for it, as we already have too much stuff in that module.  As far as I can see, it's only called from `Command`, so keeping it there is probably OK.  If it does need to be moved for some reason, let's see what's a better place for it.

Reply:
OK, no problem -- I didn't love it either, I was just trying to make minimally-invasive changes that made sense in the existing module hierarchy, but I'm happy to add this new module.

Reply:
dd8f8daa